### PR TITLE
Home page styles and landing page fixes

### PR DESCRIPTION
**Pull request by MilStancheva**
- Stats styles
- Leaderboard styles  and responsive in progress
- Club details styles and responsive in progress
- Challenge styles 
- Footer fixes
- Landing page paddings fixes

### CODE REVIEW

File: HeroesCup.Modules/ClubsModule/Services/ClubsService.cs [hunk 1]
```diff
@@ -229,5 +229,10 @@ public IEnumerable<Club> GetAllClubs()
             return this.dbContext.Clubs
                 .Include(c => c.Heroes);
         }
+
+        private string TrimInputName(string name)
+        {
+            return name.Trim(new char[] { '\"', '\'', '.', ' ', '“', '”' });
```
Comment (adautev) [inline HeroesCup.Modules/ClubsModule/Services/ClubsService.cs:235]: Test?

File: HeroesCup.Web/Models/Regions/Footer.cs [hunk 2]
```diff
@@ -5,9 +5,6 @@ namespace HeroesCup.Models.Regions
 {
     public class Footer
     {
-        [Field]
-        public HtmlField UpperContentLeft { get; set; }
```
Comment (adautev) [inline HeroesCup.Web/Models/Regions/Footer.cs:9]: Lets give the content a more logical name, not related to its position.

File: HeroesCup.Modules/ClubsModule/Services/ClubsService.cs (hunk 1 above)
Comment (MilStancheva) [reply HeroesCup.Modules/ClubsModule/Services/ClubsService.cs]: Thank you! I've changed this method to a string extension. Unit tests are added.

File: HeroesCup.Web/Models/Regions/Footer.cs (hunk 2 above)
Comment (MilStancheva) [reply HeroesCup.Web/Models/Regions/Footer.cs:9]: Of course! The names are changed according to the region purpose.

**Pull request merged**